src/scene_server/topo_server/service/inst_module.go

```diff
@@ -144,66 +135,68 @@ func (s *Service) CheckIsBuiltInModule(kit *rest.Kit, moduleIDs ...int64) errors
 			},
 		},
 	}
-	rsp, err := s.Engine.CoreAPI.CoreService().Instance().ReadInstance(kit.Ctx, kit.Header,
-		common.BKInnerObjIDModule, qc)
-	if nil != err {
-		blog.Errorf("[operation-module] failed read module instance, err: %s, rid: %s", err.Error(), kit.Rid)
+	rsp, err := s.Engine.CoreAPI.CoreService().Instance().CountInstances(kit.Ctx, kit.Header, common.BKInnerObjIDModule,
+		input)
+	if err != nil {
+		blog.Errorf("failed read module instance, err: %s, rid: %s", err.Error(), kit.Rid)
 		return kit.CCError.CCError(common.CCErrCommHTTPDoRequestFailed)
```

Comment:
```suggestion
		return err
```
这种return kit.CCError.CCError(common.CCErrCommHTTPDoRequestFailed)都检查一下改成直接返回err